packages/one-app-bundler/webpack/module/webpack.client.js

```diff
@@ -36,6 +36,7 @@ const packageRoot = process.cwd();
 const { packageJson } = readPkgUp.sync();
 const { version, name } = packageJson;
 
+const holocronModuleName = `_${name.replace(/-/g, '_')}`;
```

Comment:
Why are we replacing the dashes?

Reply:
If used `hyphens` Terser plugin throws following error. 

`Unexpected token: operator (-)`